### DISCUSSION TITLE
Should I release the actual release version (no alpha,beta,or prerelease) today?

### DISCUSSION THREAD

**Discussion by mojidev-py** [mojidev-py/discmoji#22] [General]
Title, and i will upload the full release to pypi as well.

**Comment by XenonPy** [comment]
I think you should release a beta... so not a prerelease, but wait a bit before going production. Maybe 0.3?

**Reply by mojidev-py** [reply]
sure. i was also thinking that, so i uploaded the beta release to pypi

**Reply by XenonPy** [reply]
yay

**Reply by XenonPy** [reply]
also, I will be able to accept the PyPi author invite in like 3 hours

**Reply by mojidev-py** [reply]
got it, and i'll try to setup a auto pypi upload when a release is made, but i need to change some stuff first

**Reply by XenonPy** [reply]
okie sounds good

**Reply by mojidev-py** [reply]
pypi autorelease is set up! just make a release and it should start the action

**Reply by XenonPy** [reply]
Yay!
